target/linux/ath79/dts/qca9531_glinet_gl-s200-nor-nand.dts

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "qca9531_glinet_gl-s200.dtsi"
+
+/ {
+	compatible = "glinet,gl-s200-nor-nand", "qca,qca9531";
+	model = "GL.iNet GL-S200 (NOR/NAND)";
+};
+
+&nor_partitions {
+	partition@60000 {
+		label = "kernel";
```

Comment:
Do you store the kernel in this partition for NAND systems?
I think you should also add `compatible = "denx,uimage";` here.

Reply:
No, stored in nand